biocrnpyler/components/basic.py

```diff
@@ -281,6 +656,18 @@ def update_species(self) -> List[Species]:
         return species
 
     def update_reactions(self) -> List[Reaction]:
+        """Generate binding and unbinding reactions for the complex.
+
+        Uses the 'binding' mechanism to generate reactions for complex
```

Comment:
Similar to the previous comment, I am not sure if the component "generates a reaction". The code below gets the mechanism to associate it with the component and then the mechanism generates the reactions. I understand that this is perhaps minor but it's also core to how biocrnpyler was designed.

Reply:
Also: do we want to use similar wording for `update_species`?

Reply:
I like either the first or the second possibilities you listed out: ""Call 'binding' mechanism to generate component reactions."  or "Use...."

and yes, we might similarly re-word `update_species` in the components.